cli/cmd/compose/up.go

```diff
@@ -32,6 +33,8 @@ import (
 	"github.com/spf13/cobra"
 )
 
+const environmentFileLabel = "com.docker.compose.project.environment_file"
```

Comment:
why not define this in https://github.com/docker/compose-cli/blob/main/api/compose/tags.go with other labels?

Reply:
Done